src/meds_torch/data/components/multiwindow_pytorch_dataset.py

```diff
@@ -14,6 +14,131 @@
 from meds_torch.data.components.pytorch_dataset import PytorchDataset
 
 
+def fuse_window_data(windows_data: dict, windows_to_fuse: list[str], fused_window_name: str) -> dict:
```

Comment:
It might be worth somewhere having a definition of a "window" either in a module level docstring for this file or in a separate markdown doc... I'm not totally sure, but worth a thought.